.github/workflows/ci.yml

```diff
@@ -97,14 +97,18 @@ jobs:
           # the JSON strings and later use fromJSON to decode them.
           echo "pr-title=$(jq '.title' ${PR_JSON})" >> "${GITHUB_OUTPUT}"
           echo "pr-body=$(jq '.body' ${PR_JSON})" >> "${GITHUB_OUTPUT}"
-
```

Comment:
(for threading)

I read again on the original discussion https://github.com/openxla/iree/issues/10042 about the trailers vs. labels. It seems like we thought about "it's a little intrusive to edit other's description for benchmarks", but didn't have a follow-up on it.

Reply:
I've been thinking of "running benchmarks" and "running builds/tests" as wildly different uses of presubmit, so I'd be fine with them using different mechanisms for triggering. The `skip-ci` trailer feels like a property of the PR that the author can add at creation time, while `benchmarks: all` as a label is something that authors or reviewers should be able to easily add at any point.

Reply:
Agree. I think the issue is how to support the extra benchmark sets (large models, ...) that we don't want to run by default. Having a label for each benchmark set doesn't sound very scalable. But I think benchmark labels provide more benefits in the daily use cases than its disadvantages.

So for we probably can just have benchmark labels for some default benchmark sets (x86_64, cuda, android). And if people want to trigger some extra benchmarks, they use a trailer like `extra-benchmarks` to enable those. It's confusing but only in rare use cases.

For the benchmark filter, right now I think it over-complicates the system and I don't see its usage in the near future. Well-organized benchmark presets should be enough to handle most of the cases.